go/vt/vtgate/planbuilder/operators/operator.go

```diff
@@ -14,13 +14,31 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// Package operators contains the operators used to plan queries.
+/*
+The operators go through a few phases while planning:
+1. Logical
```

Comment:
That's a nice write up

Reply:
I'll add info about horizons in the op tree as well